lib/presentation/screens/weather_screen.dart

```diff
@@ -22,12 +25,22 @@ class _WeatherScreenState extends State<WeatherScreen> {
 
   void _closeWeather() => Navigator.of(context).pop();
 
+  Future<void> _showErrorDialog(String message) => showDialog<void>(
+    context: context,
+    builder: (context) {
+      return ErrorDialog(
+        title: 'エラーが発生しました',
+        message: message,
+        onOkPressed: () => Navigator.of(context).pop(),
```

Comment:
[reviw-info]
ご存知かもしれませんが、↓のような書き方も可能です☺️


```suggestion
        onOkPressed: Navigator.of(context).pop,
```